README.md

```diff
@@ -143,9 +143,9 @@ artifacts.
 The major areas are
  
  * core - this is where the t-digest and unit tests live
- * docs - the main paper and auxillary proofs live here
+ * docs - the main paper and auxiliary proofs live here
  * benchmarks - this is the code that tests the speed of the digest algos
- * quality - this is the code that generates and analyzes accuracy information
+ * quality - this is the code that generates and analyzes accuracy of information
```

Comment:
I don't agree with this. the code analyzes information about accuracy. The information is the output of the test, not hte information contained in the t-digest.